descriptors/annotators/io/LoadPointCloud.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<taeDescription xmlns="http://uima.apache.org/resourceSpecifier">
+  <frameworkImplementation>org.apache.uima.cpp</frameworkImplementation>
+  <primitive>true</primitive>
+  <annotatorImplementationName>rs_loadPointCloud</annotatorImplementationName>
+  <analysisEngineMetaData>
+    <name>LoadPointCloud</name>
+    <description>Load Point clouds from files.</description>
+    <version>1.0</version>
+    <vendor/>
+    <configurationParameters>
+        <configurationParameter>
+            <name>folder</name>
+            <type>String</type>
+            <multiValued>false</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+        <configurationParameter>
+            <name>filenames</name>
+            <type>String</type>
+            <multiValued>true</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+    </configurationParameters>
+    <configurationParameterSettings>
+        <nameValuePair>
+            <name>folder</name>
+            <value>
+                <string>/home/anindex/</string>
+            </value>
+        </nameValuePair>
+        <nameValuePair>
+            <name>filenames</name>
+            <value>
+              <array>
```

Comment:
I am not sure we need this. Or rather it should be optional. The way I would go about this is that we have the option of specifying files or specifying folders. If it's a file it's straight forward. If it's a folder the annotator looks for the pcd (and maybe not just) files in the path provided, reads them one by one and has an option to loop.